.travis.yml

```diff
@@ -1,23 +1,18 @@
 language: julia
 julia:
   - nightly
-  - 0.6
+  - 1.0
 os:
   - linux
 branches:
   only:
     - master
 notifications:
   email: false
-git:
-  depth: 99999999
-before_install:
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-script:
-  - julia -e 'Pkg.clone(pwd());
-              Pkg.test("Glob", coverage=true)'
 after_success:
-  - julia -e 'cd(Pkg.dir("Glob"));
+  - julia -e 'using Pkg;
+              import Glob;
+              cd(dirname(dirname(pathof(Glob))));
```

Comment:
we don't want to import Glob into this process (the deprecation message is wrong), but I think `dirname(dirname(pathof(Glob))))` is supposed to be literally `pwd()`?

Reply:
😄 Cool, sounds good! Thanks